code-studio/package.json

```diff
@@ -39,7 +39,8 @@
     "build-css:dist": "npm run lint-css && npm run clean-css && scripts/build-css.js --dist",
     "build-js": "scripts/build-js.js",
     "build-js:dist": "npm run lint-js && npm test && npm run clean-js && scripts/build-js.js --dist && scripts/uglify-js.js",
-    "start": "scripts/build-js.js --watch"
+    "start": "scripts/build-js.js --watch",
+    "test-audio": "http-server . &&  echo Open your browser to http://127.0.0.1:8080/test/audio/audio_test.html"
```

Comment:
The `echo` command here isn't actually working, and you don't see it because the `http-server` process doesn't run in the background.  I mean, you see it in the command, but it's weird to have it there inoperative.

How about this?
```
"test-audio": "echo \"Open your browser to http://127.0.0.1:8080/test/audio/audio_test.html\" && http-server ."
```